### DISCUSSION TITLE
Stablizing Training Metrics

### DISCUSSION THREAD

**Discussion by sondosaabed** [community/community#123518] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hey Githubbers, I'm not sure if this is the right discussion to ask the question if not let me know which space
I' training a deep learning model (Deep CNN architecture) the split rule that I am using is 80:10:10 train:test: validation
14000:2000:2000 instances

The following curves show the learning metrics, however, it doesn't start smoothly but it follows the trend of good fitting and converges. What could be the reason for the fluctuation before the 20th epoch? and how do I make a smoother curve? Note that I'm using dropout regularization and batch normalization techniques I am also using LR schedule 

![329816393-4876ca10-0108-463d-a775-cdf7d81f18bb](https://github.com/community/community/assets/65151701/d5e37ab3-a786-4bfd-9902-6db6df5f1558)

**Comment by sondosaabed** [comment, marked as answer]
I think I could ask the question here right?
https://github.com/orgs/community/discussions/categories/programming-help
